third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -443,6 +443,20 @@ int PjRtComputationClient::GetNumProcesses() const {
   return max_process_index + 1;
 };
 
+std::map<std::string, xla::PjRtDeviceAttribute>
```

Comment:
This should be `std::map<std::string, DeviceAttribute>` to match the signature in the headers (although they're technically the same thing)